frame/curve-amm/src/tests.rs

```diff
@@ -229,3 +262,126 @@ fn exchange_test() {
 		assert!(65000 - Tokens::balance(MockCurrencyId::USDC, &CHARLIE) < 10);
 	});
 }
+
+proptest! {
+	#![proptest_config(ProptestConfig::with_cases(10000))]
+
+	#[test]
+	fn proptest_exchange(alice_balance in 1..u32::MAX,
```

Comment:
good,so test to test that curve is balanced would be awesome (kind of if make inbalance, interset grows fast or like)

Reply:
sure, I have noted this. I will try it in next PR.